about.html

```diff
@@ -1 +1,187 @@
+            </div>
+            <div class="col-xs-8">
+              Hogwart's College
+            </div>
+          </div>
+          <div class="row extra-line-height">
+            <div class="col-xs-4">
+              <strong>Hometown:</strong>
+            </div>
+            <div class="col-xs-8">
+              Godrick's Hollow, England
+            </div>
+          </div>
+          <div class="row extra-line-height">
+            <div class="col-xs-4">
+              <strong>Currently lives:</strong>
+            </div>
+            <div class="col-xs-8">
+              Godrick's Hollow, England
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-xs-12">
+              <h3>Contact Information</h3>
+            </div>
+          </div>
+          <div class="row extra-line-height">
+            <div class="col-xs-4">
+              <strong>Email:</strong>
+            </div>
+            <div class="col-xs-8">
+              harry_potter@hogwarts.edu
+            </div>
+          </div>
+          <div class="row extra-line-height">
+            <div class="col-xs-4">
+              <strong>Telephone:</strong>
+            </div>
+            <div class="col-xs-8">
+              555-123-4567
+            </div>
+          </div>
+        </div>
+        <div class="col-md-5 col-md-offset-1">
+          <div class="row">
+            <div class="col-md-12">
+              <h3>Words to live by</h3>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-md-12">
+              <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+              tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+              quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+              consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+              cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+              proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-md-12">
+              <h3>About Me</h3>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-md-12">
+              <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+              tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+              quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+              consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+              cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+              proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+            </div>
+          </div>
+        </div>
```

Comment:
I really like three things about your code.
1. You're really good with the bootstrap classes. I was doing way more manual css than necessary bc I didn't fully appreciate bootstrap's capabilities. Your code brought those to light.

2. Your code is very well organized and consistent. You don't have really any unnecessary html elements. On the other hand, when I was coding the html, I found myself leaning toward bandaid fixes - turns into spaghetti html really quick.

3. You CSS is extremely targeted and non-repetitive.